### PR TITLE
107 Add EditCommand, EditCommandTest, EditCommandParser, HelpCommand, HelpCommandTest

### PR DESCRIPTION
**Pull request by NgChunMan**

### CODE REVIEW

File: src/main/java/seedu/address/logic/newcommands/EditCommand.java [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+            AbsolutePath absolutePath = currPath.resolve(this.relativePath);
+            if (absolutePath.isGroupDirectory()) {
+                ChildOperation<Group> target = StateManager.rootChildOperation(root);
+                GroupId groupId = absolutePath.getGroupId().get();
+                if (!target.hasChild(groupId)) {
+                    throw new CommandException(ERROR_MESSAGE_NO_SUCH_GROUP);
+                }
+                Group groupToEdit = target.getChild(groupId);
+                Group editedGroup = createEditedGroup(groupToEdit, this.editGroupDescriptor);
+                target.deleteChild(groupId);
```
Comment (mingyuanc) [inline src/main/java/seedu/address/logic/newcommands/EditCommand.java:152]: Good job, you used the class correctly

File: src/main/java/seedu/address/logic/newcommands/EditCommand.java [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+        return null;
+    }
+
+    /**
+     * Represents the descriptor for editing the details of a student in ProfBook. The descriptor contains fields for
+     * updating the student's name, phone, email, address, and id. It helps to track which fields have been edited by
+     * the user.
+     * An instance of this class is used within the {@code EditCommand} to specify the details to be edited.
+     */
+    public static class EditStudentDescriptor {
```
Comment (mingyuanc) [inline src/main/java/seedu/address/logic/newcommands/EditCommand.java:233]: Reccomand storing this in another file? Unless you have a good reason to keep this as a static class

File: src/main/java/seedu/address/logic/newcommands/EditCommand.java [hunk 3]
```diff
@@ -0,0 +1,497 @@
...
+                    .toString();
+        }
+    }
+
+    /**
+     * Represents the descriptor for editing the details of a group in ProfBook. The descriptor contains fields for
+     * updating the group's name and id. It helps to track which fields have been edited by the user.
+     * An instance of this class is used within the {@code EditCommand} to specify the details to be edited.
+     */
+    public static class EditGroupDescriptor {
```
Comment (mingyuanc) [inline src/main/java/seedu/address/logic/newcommands/EditCommand.java:398]: Likewise as above comment

**Review by mingyuanc — APPROVED** [review]
Next time when ur creating two different feature in two different branches, you shouldnt copy the code over, just keep it seperate

**Review by NereusWB922 — APPROVED** [review]

**Pull request merged**